modules/spo-applying-profiles.adoc

```diff
@@ -0,0 +1,180 @@
+// Module included in the following assemblies:
+//
+// * security/security_profiles_operator/spo-seccomp.adoc
+// * security/security_profiles_operator/spo-selinux.adoc
+
+ifeval::["{context}" == "spo-seccomp"]
+:seccomp:
+:type: seccomp
+:kind: SeccompProfile
+endif::[]
+ifeval::["{context}" == "spo-selinux"]
+:selinux:
+:type: SELinux
+:kind: SelinuxProfile
+endif::[]
+
+:_content-type: PROCEDURE
+[id="spo-applying-profiles_{context}"]
+= Applying {type} profiles to a pod
+
+Create a Pod to apply one of the created profiles.
+
+ifdef::selinux[]
+For {type} profiles, the namespace must be labelled to allow link:https://kubernetes.io/docs/concepts/security/pod-security-standards/[privileged] workloads.
+
+endif::[]
+.Procedure
+
+ifdef::seccomp[]
+. Create a Pod object that defines a `securityContext`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: test-pod
+spec:
+  securityContext:
+    seccompProfile:
+      type: Localhost
+      localhostProfile: operator/my-namespace/profile1.json
+  containers:
+    - name: test-container
+      image: nginx
+----
+
+. View the profile path of the `seccompProfile.localhostProfile` attribute by running the following command:
++
+[source,terminal]
+----
+$ oc -n my-namespace get seccompprofile profile1 --output wide
+----
++
+.Example output
+[source,terminal]
+----
+NAME       STATUS   AGE   SECCOMPPROFILE.LOCALHOSTPROFILE
+profile1   Active   14s   operator/my-namespace/profile1.json
+----
+
+. View the path to the localhost profile by running the following command:
++
+[source,terminal]
+----
+$ oc get sp profile-allow-unsafe --output=jsonpath='{.status.localhostProfile}'
```

Comment:
If I'm following along with the example, should this be `my-profile` or `profile1`?